src/test/java/org/broadinstitute/hellbender/utils/StringSplitSpeedUnitTest.java

```diff
@@ -0,0 +1,197 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.tribble.util.ParsingUtils;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.annotations.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * A class to test the speed of different string.split implementations.
```

Comment:
Add a note that these tests are disabled by default.

Reply:
Fixed!